operator/pkg/manifest/installer.go

```diff
@@ -60,6 +56,7 @@ import (
 	"istio.io/istio/operator/pkg/util"
 	pkgversion "istio.io/istio/operator/pkg/version"
 	binversion "istio.io/istio/operator/version"
+	"istio.io/istio/pilot/pkg/model" // For GCP auth functionality.
```

Comment:
why is this for auth?

Reply:
This was left over after the auto-format did something weird. Fixed.